doc/reference/devices_gpu.md

```diff
@@ -51,8 +51,105 @@ Add a specific GPU from the host system as a `physical` GPU device to an instanc
 
     lxc config device add <instance_name> <device_name> gpu gputype=physical pci=<pci_address>
 
+Add a specific GPU from the host system as a `physical` GPU device to an instance using the [Container Device Interface](https://github.com/cncf-tags/container-device-interface) (CDI) notation.
```

Comment:
Does this support only containers? If so we should say :

> Add a specific GPU from the host system as a `physical` GPU device to an *container*

Reply:
Where is `fq_CDI_name` defined in the docs? How can the end user know what that is?

Reply:
fixed